kong.conf.default

```diff
@@ -173,6 +173,15 @@
                                  #   It runs DB-less and receives configuration
                                  #   updates from a control plane node.
 
+#cluster_mtls = shared           #
+                                 # Valid values to this setting are:
+                                 #
+                                 # - `shared`: use a shared certificate/key
+                                 #   pair specified with the `cluster_cert`
+                                 #   and `cluster_cert_key` settings.
```

Comment:
We should add "In this mode the CP and DP nodes are required to present the same certificate to establish mTLS connection with each other." to explain what this setting means for the user.